src/app/beer_garden/api/http/base_handler.py

```diff
@@ -269,11 +273,39 @@ def validate_or_raise(
             The result of applying the method to the data
 
         Raises:
-            ValidationError: The supplied data failed to validate against the schema
+            BadRequest: The supplied data failed to validate against the schema
         """
-        schema = schema(strict=True) if not many else schema(strict=True, many=True)
+        schema = schema(strict=True, many=many)
 
         try:
-            return getattr(schema, method)(arg).data
+            return schema.loads(arg).data if from_string else schema.load(arg).data
         except MarshmallowValidationError as mmve:
-            raise ValidationError(reason=str(mmve))
+            raise BadRequest(reason=str(mmve))
+
+    def format_response(
```

Comment:
I'm not married to this name.

Reply:
This name seems okay.  This had me thinking about a previous conversation we had somewhere where I think we you trying to detect whether `many=True` should be set or not and it didn't feel appropriate to me in that instance.  Here it feels like that concept could make sense if you wanted to apply it.  Basically, `format_response` could have `many` be totally optional and if it's not set, you set it to true if the received `arg` is iterable or some such thing.  Definitely not necessary, but I remembered you attempting to do this somewhere else so I figured I'd bring it up as an option.